pkg/http/api.go

```diff
@@ -406,7 +406,7 @@ func (a *api) onGetSecret(c *routing.Context) error {
 	}
 
 	if resp.Data == nil {
```

Comment:
Since `resp` is a pointer ref,  can this be `if resp == nil || resp.Data == nil`?

Reply:
In this case resp is not a pointer ref:

https://github.com/dapr/components-contrib/blob/d40be01831c423aea27e09933a64e175bb3d8575/secretstores/secret_store.go#L13

which is different approach from the one used in the StateStore:

https://github.com/dapr/components-contrib/blob/d40be01831c423aea27e09933a64e175bb3d8575/state/store.go#L13 

Should we update the SecretStore component?

Reply:
Yes

Reply:
No, right. If it's a non pointer is should be fine.